src/Orchard.Web/Modules/Orchard.DynamicForms/Views/Elements/ReCaptcha.cshtml

```diff
@@ -5,7 +5,7 @@
     var element = (ReCaptcha)Model.Element;
     var publicKey = (string)Model.PublicKey;
 }
-@if (!String.IsNullOrWhiteSpace(publicKey)) {
+@if (HasText(publicKey)) {
```

Comment:
I was told to change from IsNullOrWhitespace() to HasText() by Jetski5822.
I can change it back though if you'd like.

On Sun, Jul 26, 2015, 4:47 AM Sipke Schoorstra <notifications@github.com>
wrote:

> In
> src/Orchard.Web/Modules/Orchard.DynamicForms/Views/Elements/ReCaptcha.cshtml
> <https://github.com/OrchardCMS/Orchard/pull/5573#discussion_r35491429>:
>
> > @@ -5,7 +5,7 @@
> >      var element = (ReCaptcha)Model.Element;
> >      var publicKey = (string)Model.PublicKey;
> >  }
> > -@if (!String.IsNullOrWhiteSpace(publicKey)) {
> > +@if (HasText(publicKey)) {
>
> I wonder why you're using HasText on strings? Isn't that intended to be
> used on objects other than strings?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OrchardCMS/Orchard/pull/5573/files#r35491429>.
>